package.json

```diff
@@ -83,36 +81,29 @@
         "webpack-dev-server": "^3.11.0"
     },
     "dependencies": {
-        "@aics/simularium-observables-manager": "^0.1.0",
         "@aics/simularium-viewer": "^2.9.7",
         "@ant-design/css-animation": "^1.7.3",
         "@ant-design/icons": "^4.0.6",
-        "@types/react-plotly.js": "^2.2.4",
-        "@types/react-router-dom": "^5.1.5",
-        "@types/victory": "^31.0.12",
         "antd": "^4.0.0-rc.6",
         "axios": "^0.21.1",
         "bowser": "^2.11.0",
         "classnames": "2.2.5",
         "core-js": "2.5.5",
         "lodash": "^4.17.20",
-        "momentjs": "^2.0.0",
+        "moment": "^2.0.0",
```

Comment:
Apparently this can be `momentjs` or `moment` and both works? Super confusing. But `moment` matches the import statement we use in this project